cmd/dep.go

```diff
@@ -102,6 +106,65 @@ With these commands, a user can 'import "kspkg://nginx1"', and
 'import "kspkg://nginx2"' with no conflict.`,
 }
 
+var depDescribeCmd = &cobra.Command{
+	Use:   "describe [<registry-name/]<package-name>",
```

Comment:
`/` should be `>` in `<registry-name/`

Reply:
I'm changing it to `[<registry-name>/]`, because the `/` is not optional if you include `registry-name`.